zigcc.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
```

Comment:
Can we move these files to `${CMAKE_SOURCE_DIR}/bindings/zig`?

Reply:
moved to `${CMAKE_SOURCE_DIR}/bindings/zig/tools`